app/assets/stylesheets/encounters.scss

```diff
@@ -112,31 +112,31 @@
 	position: relative;
 	> span.dropdown-button {
 		position: absolute;
-		right: 8px;
-		top: 8px;
+		right: .5rem;
+		top: .5rem;
 
 		button.arrow-button {
-			height: 30px;
-			width: 30px;
-			line-height: 30px * .80;
+			height: 1.875rem;
+			width: 1.875rem;
+			line-height: 1.5rem;
 		}
 	}
 
 	>.dropdown-content {
-		padding: 0px 10px;
+		padding: 0rem .625rem;
 	}
 }
 
 .contacts-show {
-	padding-inline-start: 0px;
+	padding-inline-start: 0rem;
 
 	.contact-show {
 		clear: both;
-		margin-bottom: 5px;
+		margin-bottom: 0.313rem;
 
 		>.dropdown {
 			transition: all 0.3s ease-in-out;
-			border-bottom: 1px solid $color-green-dark;
+			border-bottom: .063rem solid $color-green-dark;
```

Comment:
With stuff like borders, does it ever make sense to leave it in pixels with the idea that maybe it shouldn't scale up? Or is the idea that someone who needs the text to be larger would also need the borders to be wider in order for them to be visible?

Reply:
i think it's totally fine to leave it at 1px here, and 1px values.

what you're really saying is "render the thinnest line possible" (not accounting for retina-screens, but still that's basically what it means).

i think the "scaling" issue is something that's a recurring theme so i'll talk about that separately.